src/uwtools/logging.py

```diff
@@ -0,0 +1,26 @@
+"""
+Logging support.
+"""
+
+import logging
+import os
+import sys
+
+
+def setup_logging(quiet: bool = False, verbose: bool = False) -> None:
+    """
+    Set up logging.
+
+    :param quiet: Supress all logging output.
+    :param verbose: Log all messages.
+    """
+    if quiet and verbose:
+        print("Specify at most one of 'quiet' and 'verbose'", file=sys.stderr)
+        sys.exit(1)
+    kwargs: dict = {
+        "datefmt": "%Y-%m-%dT%H:%M:%S",
+        "format": "[%(asctime)s] %(levelname)8s %(message)s",
+        "level": logging.DEBUG if verbose else logging.INFO,
+        **({"filename": os.devnull} if quiet else {}),
+    }
+    logging.basicConfig(**kwargs)
```

Comment:
Here's how this looks in practice:
```
>>> from uwtools.logging import setup_logging
>>> setup_logging()
>>> import logging
>>> logging.info("Some info")
[2023-08-22T00:41:14]     INFO Some info
>>> logging.critical("Oh no!")
[2023-08-22T00:41:23] CRITICAL Oh no!
```
If the `filename` argument to `basicConfig()` is not set, Python's root logger defaults to logging to `stderr`, which is what we want.

Reply:
Just to make sure I'm reading the correctly...we lose the coloring scheme that was set up in the class that was removed, is that right?

Reply:
Yes, but we could add that, or something similar, back later if we find that we need it. It seemed to me maybe premature at this point. I'm happy to commit to adding color-coding if/when we want it -- including now.

IMO colors can be nice in human-facing tools, but unwelcome in automated contexts where the output might be captured/parsed, as the ANSI color codes can complicate e.g. regex processing. For just about any tool with colorized output, animated progress bars, etc., there are tickets from disgruntled users asking for ways to disable it. So it should at least be behavior that can be disabled -- which I believe was indeed already the case.